lib/iex/endpoints/key_stat.rb

```diff
@@ -0,0 +1,14 @@
+module IEX
+  module Endpoints
+    module KeyStat
+      def key_stat(symbol, stat, options = {})
+        result = get("stock/#{symbol}/stats/#{stat}", { token: publishable_token }.merge(options))
+        raise IEX::Errors::StatNotFoundError.new(stat, result) if result.is_a?(Hash)
```

Comment:
Should this be `unless result.is_a?(Number)` or similar?

Reply:
I wasn't able to find an exhaustive list of types IEX currently returns. From experience I know it can return numbers, strings (for strings and dates), and booleans. Testing for one type I know indicates an invalid stat is simpler and more permissive of future changes with IEX's API. 😅 For all I know IEX will return some stat as a tuple encoded as an array and someone would have a bad day debugging because the requested stat is valid.